setup.cfg

```diff
@@ -35,10 +35,11 @@ test =
     pytest-cov
     nbconvert>=6.4.5
 visualtest =
-    playwright
-    pytest-playwright
+    playwright==1.41.2
+    pytest-playwright==0.5.2
     pytest-mpl
-    solara[pytest]<1.29
+    pytest-ipywidgets[all]
+    solara-ui @ https://github.com/widgetti/solara/archive/refs/heads/fix_matplotlib_backend_keyerror.zip
```

Comment:
Something therein is pinning jupyter-client 7.x, jupyterlab 3.x and notebook 6.x now, among others.
But for visualtest only we probably can/have to live with that for now.